packages/components/package.json

```diff
@@ -51,6 +53,7 @@
     "marked": "^1.2.3"
   },
   "peerDependencies": {
+    "nebenan-components": "^9.3.2",
```

Comment:
I removed nebenan-components in a different PR - did you merge the newest master branch?

All those packages added here should be gone